src/app/client/src/app/modules/cbse-program/services/cbse-program/data.ts

```diff
@@ -132,19 +132,34 @@ export let themeObject = {
           'type': 'json'
       },
       {
-          'id': 'org.sunbird.questionunit.quml',
+          'id': 'org.sunbird.questionunit.quml.plugin_js',
```

Comment:
@vaibhavbhuva Please correct the naming convention for plugin
cc: @pallakartheekreddy